gemonlinedb/parseDef.py

```diff
@@ -0,0 +1,270 @@
+#! /usr/bin/env python2
+
+import xml.etree.ElementTree as ET
+
+def _checkedJsonGet(json, fieldName, targetType):
+    """
+    Returns a value from a json dictionary only if it has the right type
+    """
+    assert(fieldName in json)
+    assert(type(json[fieldName]) == targetType)
+    return json[fieldName]
+
+def _toCamelCase(snake_string, capitalizeFirstLetter=False):
```

Comment:
I see, probably this would be where implementation of the "almost" camel case from the comment above would give the most trouble...

Reply:
Fixed by supporting manual override of C++ names in the JSON file.